checks.d/openstack.py

```diff
@@ -167,7 +192,11 @@ def get_stats_for_single_network(self, network_id):
         resp = self._make_request_with_auth_fallback(url, headers)
 
         net_details = resp.json()
-        service_check_tags = ['network:{0}'.format(network_id)]
+        service_check_tags = [
+            'network:{0}'.format(network_id),
+            'network_name:{0}'.format(net_details.get('network', {}).get('name')),
+            'tenant_id:{0}'.format(net_details.get('network', {}).get('tenant_id'))
```

Comment:
I can't remember why I did it this way. I may have just copy/pasted from a different location. I'll clean it up.